layers/meta-balena-jetson/recipes-containers/balena/balena_git.bbappend

```diff
@@ -0,0 +1 @@
+FILES_COMPRESS=""
```

Comment:
Correct, change is not needed anymore. Dropping it since it was fixed since upx3.94 and I see no issues with containers. https://github.com/balena-os/meta-balena/commit/1d70610